sample-extension/EdFi.Ods.Extensions.Sk/EdFi.Ods.Extensions.Sk.csproj

```diff
@@ -57,10 +60,4 @@
     </EmbeddedResource>
     <EmbeddedResource Include="assemblyMetadata.json" />
   </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\..\..\Ed-Fi-ODS\Application\EdFi.Ods.Api\EdFi.Ods.Api.csproj" />
-    <ProjectReference Include="..\..\..\Ed-Fi-ODS\Application\EdFi.Ods.Common\EdFi.Ods.Common.csproj" />
-    <ProjectReference Include="..\..\..\Ed-Fi-ODS\Application\EdFi.Ods.Security\EdFi.Ods.Security.csproj" />
```

Comment:
we don't need this EdFi.Ods.Security package reference . Asking this understand why not reference again